### ISSUE TITLE
Max jitter value isn't max jitter in results list

### ISSUE THREAD

**Issue by jwbensley** [jwbensley/Etherate#51]
##### ISSUE TYPE

- [X] Bug Report
- [ ] Enhancement/Feature Request
- [ ] Question

##### VERSION DETAILS
```
Etherate version 1.17 2018-04
```

##### SUMMARY
Results from Tx host below. The output says `Min/Max jitter during test: 0.000000017s/0.000006009s` but the highest jitter value recorded was `0.000122755s`.

##### STEPS TO REPRODUCE
sudo ./etherate -I ens2f0 -Q 1000 1000

```
No.	rtt		Jitter
1:	0.000122755s	0.000122755s
2:	0.000064407s	0.000058348s
3:	0.000065850s	0.000001443s
4:	0.000071859s	0.000006009s
5:	0.000066288s	0.000005571s
6:	0.000062803s	0.000003485s
7:	0.000066670s	0.000003867s
8:	0.000063426s	0.000003244s
9:	0.000065757s	0.000002331s
10:	0.000067083s	0.000001326s
11:	0.000067066s	0.000000017s
12:	0.000065880s	0.000001186s
13:	0.000067321s	0.000001441s
14:	0.000063679s	0.000003642s
15:	0.000064127s	0.000000448s
16:	0.000064932s	0.000000805s
17:	0.000065436s	0.000000504s
18:	0.000063706s	0.000001730s
19:	0.000068546s	0.000004840s
20:	0.000066065s	0.000002481s
21:	0.000062928s	0.000003137s
22:	0.000066641s	0.000003713s
23:	0.000067589s	0.000000948s
24:	0.000065886s	0.000001703s
25:	0.000062733s	0.000003153s
26:	0.000064417s	0.000001684s
27:	0.000064453s	0.000000036s
28:	0.000065138s	0.000000685s
29:	0.000066881s	0.000001743s
30:	0.000063702s	0.000003179s
Link quality test complete
Test frames transmitted: 30
Test frames received: 30
Non test or out-of-order frames received: 0
Number of timeouts: 0
Min/Max rtt during test: 0.000062733s/0.000071859s
Min/Max jitter during test: 0.000000017s/0.000006009s
Ending test on Sat Apr 14 12:38:29 2018
```

**Comment by jwbensley** [comment]
Thanks same is true for RTT max.

**Comment by jwbensley** [comment]
Fixed in 1.18.
